addons/medical_engine/script_macros_medical.hpp

```diff
@@ -3,6 +3,12 @@
 #define ALL_SELECTIONS ["head", "body", "hand_l", "hand_r", "leg_l", "leg_r"]
 #define ALL_HITPOINTS ["HitHead", "HitBody", "HitLeftArm", "HitRightArm", "HitLeftLeg", "HitRightLeg"]
 
+// Damage threshold above which fatal organ damage can occur
+#define ORGAN_DAMAGE_THRESHOLD 1
+// Chance to hit heart based on ratio of 70kg (approx. 70L) body to 70mL stroke volume of heart
+// Assuming torso is 50% of the body volume (35L)
+#define HEART_HIT_CHANCE 0.005
```

Comment:
Shouldn't this be 70/(35*1000) = 0.002? You have hit the torso already so, the 35L is your 100%

Reply:
It's a pitty we cannot really check were the hit is relative to the centre of masses of the given unit...

Reply:
I rounded it up to the nearest half percent because it's such a small chance already.

Yeah, even more ideally units would have organs modelled inside where we could detect hits on them